app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  before_filter :devise_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def devise_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) {
+      |u| u.permit(:email, :password, :password_confirmation, :image) }
```

Comment:
Expression at 10, 71 should be on its own line.<br>Block argument expression is not on the same line as the block start.